math/pi/montecarlopi.go

```diff
@@ -22,6 +22,6 @@ func MonteCarloPi(randomPoints int) float64 {
 			inside += 1
 		}
 	}
-	pi := float64(inside) / float64(randomPoints) * float64(4)
+	pi := float64(inside) / float64(randomPoints) * 4
```

Comment:
```suggestion
	pi := float64(inside) / randomPoints * 4
```
Dividing the `float64` number with `randomPoints` will automatically result in a `float64` number.

Reply:
Oh yeah, you're right

Reply:
Pardon me, I forgot golang does not have implicit type conversions, I have fixed that in the latest commit.